dist-build/android-x86_64.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
-export TARGET_ARCH=westmere
-export CFLAGS="-Os -march=${TARGET_ARCH}"
-NDK_PLATFORM=android-21 ARCH=x86_64 HOST_COMPILER=x86_64-linux-android "$(dirname "$0")/android-build.sh"
+export CFLAGS="-Os -march=westmere"
+NDK_PLATFORM=android-21 HOST_COMPILER=x86_64-linux-android TARGET_ARCH_DIR="${HOST_COMPILER}" "$(dirname "$0")/android-build.sh"
```

Comment:
Should `NDK_PLATFORM` be target-specific? Maybe it can be set only once in `android-build.sh`.

Reply:
I have no issues making it target specific. I left it as per the old scripts I figured the intention was to maximize legacy support but I never really researched it.

Reply:
I'm going to do one commit after I get everything sorted as some of the items might no longer be relevant once the larger layout is decided so I'm going to mark this and a few other minor items as resolved but no code change will be visible yet.